src/doc/python_scripting/quickrecipes.rst

```diff
@@ -1014,31 +1014,42 @@ Here's an example of creating a discrete color table using named colors from the
     Subset plot using the "mylevels" color table.
 
 
-Plots with special handling of color tables
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Volume Plot's special handling of its color table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:ref:`Contour <contour_plot_head>`, MultiCurve, :ref:`Volume <volume_plot_head>` and Well Bore plots all store a ``ColorControlPointList`` directly instead of being referenced via color table name.
+:ref:`Volume <volume_plot_head>` plots Color Table is stored directly as a ``ColorControlPointList`` rather than indirectly from a color table name.
 
-Contour, MultiCurve and Well Bore plots
-"""""""""""""""""""""""""""""""""""""""
+The colorControlPoints object is initialized with 5 values corresponding to the `hot` Color Table.
+The size of the colorControlPoints list can be adjusted in several ways:
+via `AddControlPoints`, `RemoveControlPoints` or `SetNumControlPoints` called on the colorControlPoints object, or via `SetColorControlPoints` called on the VolumeAttributes object.
```

Comment:
I think you want two backticks on all these also, right? And some cases below here as well?